src/model/FeaturesView.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { Fragment, useCallback } from 'react';
+import ProtvistaTrack from 'protvista-track';
+import ProtvistaManager from 'protvista-manager';
+import ProtvistaDatatable from 'protvista-datatable';
+import ProtvistaSequence from 'protvista-sequence';
+import ProtvistaNavigation from 'protvista-navigation';
+import { loadWebComponent } from '../utils/utils';
+
+type FeatureLocation = {
+  value: number;
+  modifier: string;
+};
+
+type Feature = {
+  type: string;
+  featureId: string;
+  description: string;
+  location: {
+    start: FeatureLocation;
+    end: FeatureLocation;
+  };
+};
+
+type ProtvistaFeature = {
+  type: string;
+  description: string;
+  start: number;
+  end: number;
+};
+
+type FeaturesViewProps = {
+  data: {
+    sequence: {
```

Comment:
Should all of these be optional attributes too?

Reply:
According to Leo, `sequence` is one of the only things we can assume is always there. No entry without sequence.